ffi/src/primitives.rs

```diff
@@ -133,5 +141,153 @@ primitive_derive! { u32, i32, u64, i64 }
 #[cfg(not(target_family = "wasm"))]
 primitive_derive! { u8, i8, u16, i16 }
 
-// TODO: u128/i128 is not FFI-safe. Must be properly serialized!
-primitive_derive! { u128, i128 }
+/// Ffi-safe representation of `u128`
+#[derive(Clone, Copy, Debug, Default)]
+#[repr(transparent)]
+pub struct FfiU128(FfiTuple2<u64, u64>);
```

Comment:
alignment of this struct is 8, had we used 16 elem tuple or `[u8; 16]` alignment would be `1`. What do you think about that?

Reply:
Using [u8;16] will be more convenient and fast since we have std methods for such conversion. 
Array decay to pointer shouldn't be the problem too since we using array inside struct, right? We just need to change transparent repr to repr(C). 
And since we are converting and not transmuting values alignment shouldn't be the problem.

Reply:
I don't mind either way. Personally, `[u8; 16]` tells me a lot more about what we're doing to the type.